jenkins-vars.yml

```diff
@@ -0,0 +1,29 @@
+---
+
+# jenkins variables
+project_name: docker-netbox
+external_type: github_stable
+release_type: stable
+release_tag: latest
+ls_branch: master
+repo_vars:
+  - EXT_GIT_BRANCH = 'master'
+  - EXT_USER = 'netbox-community'
+  - EXT_REPO = 'netbox'
+  - BUILD_VERSION_ARG = 'NETBOX_RELEASE'
+  - LS_USER = 'linuxserver'
+  - LS_REPO = 'docker-netbox'
+  - CONTAINER_NAME = 'netbox'
+  - DOCKERHUB_IMAGE = 'linuxserver/netbox'
+  - DEV_DOCKERHUB_IMAGE = 'lsiodev/netbox'
+  - PR_DOCKERHUB_IMAGE = 'lspipepr/netbox'
+  - DIST_IMAGE = 'alpine'
+  - MULTIARCH='true'
+  - CI='true'
+  - CI_WEB='true'
```

Comment:
The screenshots in CI don't seem to be loading. Any idea why?

Reply:
> Couple of notes here on a few things. Also the CI logs look like they have syntax errors from python. I'm not a python expert so I don't quite know what to suggest for with that.

I think the error is a Jenkins build thing since master doesn't have a Jenkensfile. Aptalca said this is normal and a bug that will be fixed in the next monthly build.

Reply:
> The screenshots in CI don't seem to be loading. Any idea why?

Not sure. Where are you supposed to be seeing the screenshots? Also, this container depends on 2 external containers, that could be why? I don't think this will past the general smoke tests.